.github/workflows/create_gh_discussion.yml

```
@@ -100,6 +105,8 @@ jobs:
                 title: title,
                 body: description
               });
+
+              return result;
```

(1/2) result が何であるか曖昧なので、discussionId を返したほうが良いと思います。

```suggestion
              return result.createDiscussion.discussion.id;
```

createDiscussion 関数の戻り値が discussionId なのはシンプルだと思います。

P.S.
JSDoc 復活させたほうが良いですかね？ ← 優柔不断奴

---

> JSDoc 復活させたほうが良いですかね？

まぁなくてもいいかな 💭 と思ったりしてます！